src/main/java/net/mcreator/ui/dialogs/AddCommonTagsDialog.java

```diff
@@ -93,15 +94,19 @@ public static void open(MCreator mcreator) {
 		callables.add(addTag(mcreator, itemTags, "planks", "minecraft", TagType.ITEMS, false));
 		callables.add(addTag(mcreator, itemTags, "flowers", "minecraft", TagType.ITEMS, false));
 		callables.add(addTag(mcreator, itemTags, "small_flowers", "minecraft", TagType.ITEMS, false));
-		callables.add(addTag(mcreator, itemTags, "small_flowers", "minecraft", TagType.ITEMS, false));
+		callables.add(addTag(mcreator, itemTags, "meat", "minecraft", TagType.ITEMS, false));

```

Comment:
Seems like you have removed small_flowers when adding meat, please add small_flowers back

Reply:
This tag is present there on the master branch twice.

Reply:
Aha, I see